docs/devel/hostpath-provisioner.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hostpath-provisioner
+  labels:
+    app: hostpath-provisioner
+spec:
+  selector:
+    matchLabels:
+      app: hostpath-provisioner
+  replicas: 1
+  revisionHistoryLimit: 0
+  template:
+    metadata:
+      labels:
+        app: hostpath-provisioner
+    spec:
+      serviceAccount: hostpath-sa
+      serviceAccountName: hostpath-sa
+      containers:
+        - name: hostpath-provisioner
+          image: mazdermind/hostpath-provisioner:latest
```

Comment:
I don't feel comfortable advertising this image in the kubevirt src tree. I'd prefer we built our own image that does the same thing so we are confident we can trust what's in the container. 

I used this image locally for myself, but I'm hesitant to officially recommend it in the kubevirt documentation.

Reply:
Sure, I am not familiar with the process of creating our own image, if you can point me at the documentation on how to do that, I will see if I can create one. Another good reason if the guy decides to pull the image, we won't have access to it anymore either.

Reply:
@davidvossel never mind I figured out how to create an image, I noticed we have a bunch of kubevirt/whatever images already, the question is, can I push my image there (kubevirt/hostpath-provisioner), or should I create my own (I have kubevirtdevelstorage right now).